bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -56,58 +82,120 @@ def tearDown(self):
         self.p2.stop()
         self.p3.stop()
 
+    def _patch_if_necessary(self, expected):
+        # no patching needed for two-digit version numbers
+        return expected
 
-class TestLatestBuilds(PatchFirefoxDesktopMixin, TestCase):
-    def test_latest_builds(self):
+
+class PatchFirefoxDesktopThreeDigitsMixin:
+    firefox_desktop_versions = GOOD_VERSIONS__THREE_DIGITS
+
+    def setUp(self):
+        self.firefox_desktop = FirefoxDesktop(json_dir=PROD_DETAILS_DIR)
```

Comment:
Note that I haven't expanded tests re the prod details, because they don't look like they will cause problems when they roll to three digits

Reply:
(I may be proven wrong, of course)